editor/src/messages/tool/tool_messages/select_tool.rs

```diff
@@ -855,8 +855,8 @@ impl Fsm for SelectToolFsmState {
 				SelectToolMessage::DragStart {
 					extend_selection,
 					remove_from_selection,
-					select_deepest,
-					lasso_select,
+					select_deepest_key,
+					lasso_select_key,
```

Comment:
renamed this so it's consistent with other similar var names and it let me use `lasso_select` as a different var in logic below